### DISCUSSION TITLE
Error Message while trying to use spaCy experimental package for CorefResolver

### DISCUSSION THREAD

**Discussion by kausikb** [explosion/spaCy#13113] [Help: Other Questions] [answered]
While using spaCy experimental package for CorefResolver, I am getting the following error, Any clue what package could be missing?

RegistryError: [E893] Could not find function 'spacy-experimental.Coref.v1' in function registry 'architectures'. If you're using a custom function, make sure the code is available. If the function is provided by a third-party package, e.g. spacy-transformers, make sure the package is installed in your environment.

Available names: spacy-legacy.CharacterEmbed.v1, spacy-legacy.EntityLinker.v1, spacy-legacy.HashEmbedCNN.v1, spacy-legacy.MaxoutWindowEncoder.v1, spacy-legacy.MishWindowEncoder.v1, spacy-legacy.MultiHashEmbed.v1, spacy-legacy.Tagger.v1, spacy-legacy.TextCatBOW.v1, spacy-legacy.TextCatCNN.v1, spacy-legacy.TextCatEnsemble.v1, spacy-legacy.Tok2Vec.v1, spacy-legacy.TransitionBasedParser.v1, spacy.CharacterEmbed.v2, spacy.EntityLinker.v2, spacy.HashEmbedCNN.v2, spacy.MaxoutWindowEncoder.v2, spacy.MishWindowEncoder.v2, spacy.MultiHashEmbed.v2, spacy.PretrainCharacters.v1, spacy.PretrainVectors.v1, spacy.SpanCategorizer.v1, spacy.Tagger.v2, spacy.TextCatBOW.v2, spacy.TextCatCNN.v2, spacy.TextCatEnsemble.v2, spacy.TextCatLowData.v1, spacy.Tok2Vec.v2, spacy.Tok2VecListener.v1, spacy.TorchBiLSTMEncoder.v1, spacy.TransitionBasedParser.v2

## My Environment

* Operating System: Windows 11 (22H2)
* Python Version Used: v3.11.15
* spaCy Version Used: v3.5.3
* Code Sinippet: 
 >from spacy_experimental.coref.coref_component import DEFAULT_COREF_MODEL
>from spacy_experimental.coref.coref_util import DEFAULT_CLUSTER_PREFIX
> doc = nlp("This is a sentence.")
> coref = nlp.add_pipe("experimental_coref")

**Comment by adrianeboyd** [comment, marked as answer]
This discussion covers a lot of the basics for using the experimental coref component: https://github.com/explosion/spaCy/discussions/13075
